### ISSUE TITLE
specextract talks about using mkacisrmf twice

### ISSUE THREAD

**Issue by DougBurke** [cxcsds/ciao-contrib#671]
This is with CIAOX but not sure it's changed much.

```
% specextract primary/acisf27294N001_evt2.fits.gz"[sky=region(ds9.reg)]" ciaox/src weight- correct+ clob+ 
Running specextract
Version: 10 February 2022

Aspect solution file primary/pcadf27294_000N001_asol1.fits found.

Bad pixel file primary/acisf27294_000N001_bpix1.fits.gz found.

Mask file secondary/acisf27294_000N001_msk1.fits found.

Setting bad pixel file 

Converting source region to physical coordinates 

Extracting src spectra 

Using mkacisrmf...

Creating src ARF 

Using mkacisrmf...

Creating src RMF 

Calculating aperture correction for src ARF 

Grouping src spectrum 

Updating header of ciaox/src.pi with RESPFILE and ANCRFILE keywords.

Updating header of ciaox/src_grp.pi with RESPFILE and ANCRFILE keywords.
```

Note that the screen output says `Using mkacisrmf...` and then `Creating src ARF` and then `Using mkacisrmf...`. Are we doing redundant work or are strange things afoot at the circle K?

**Comment by nplee** [comment]
@DougBurke There's definitely some redundancies here.  I've corrected this for 4.15, and I'm like 99.999% sure (can't check right now, internet's gone down in the area, but can see the Comcast bucket truck down the street doing work) this is also fixed in the 4.14.4 update.

**Comment by DougBurke** [comment]
With a 4.14.4 run I saw

```
Converting source region to physical coordinates 

Extracting src spectra 

Using mkacisrmf...

Creating src RMF 

Creating src ARF 

Calculating aperture correction for src ARF 

```

so as you said it looks to be fixed there
